src/language/dafnyLanguageClient.ts

```diff
@@ -61,9 +74,19 @@ export class DafnyLanguageClient extends LanguageClient {
     return this.sendRequest<ICounterExampleItem[]>('dafny/counterExample', param);
   }
 
-  public static async create(context: ExtensionContext): Promise<DafnyLanguageClient> {
+  public static argumentsToCommandLine(launchArguments: string[]): string {
+    return launchArguments.map(oneArgument =>
+      (/\s|"|\\/.exec(oneArgument))
+        ? '"' + oneArgument.replace(/\\/g, '\\\\').replace(/"/g, '\\"') + '"'
+        : oneArgument
+    ).join(' ');
+  }
+
+  public static async create(context: ExtensionContext, statusOutput: OutputChannel): Promise<DafnyLanguageClient> {
     const dotnetExecutable = await getDotnetExecutablePath();
     const launchArguments = [ getLanguageServerRuntimePath(context), ...getLanguageServerLaunchArgs() ];
+    statusOutput.appendLine(
+      `Language server arguments: ${DafnyLanguageClient.argumentsToCommandLine(launchArguments)}`);
```

Comment:
Honestly, I don't like these random line breaks IntelliJ/Rider/WebStorm does, it should at least move the closing parenthesis to a new line too. Unfortunately, I haven't found an eslint rule to enforce this yet.